frontend/app/pods/admin/teams/template.hbs

```diff
@@ -7,6 +7,7 @@
 <div class="row">
   <div class="small-5 columns">
     {{#each model as |team|}}
+      {{#if team.name}}
```

Comment:
Is it a legal case to have a nameless team? I think it should be mandatory (if not the API needs to be modified)